### PR TITLE
base_contract: Improve your OCA PR

### PR DESCRIPTION
**Pull request by alexis-via**
FIX ir.rule for odoo 13
Replace deprecated _company_default_get() by standard v13 syntax
Add domain field
Add chatter on agreement object

### CODE REVIEW

**Comment by alexis-via** [comment]
@i-vyshnevska Could you merge this ?

File: base_contract/views/agreement.xml [hunk 1]
```diff
@@ -12,33 +12,41 @@
...
     <field name="arch" type="xml">
         <form string="Agreement">
+            <sheet>
             <div class="oe_button_box" name="button_box">
-                <button name="toggle_active" type="object"
-                    class="oe_stat_button" icon="fa-archive">
-                    <field name="active" widget="boolean_button"
-                        options='{"terminology": "archive"}'/>
-                </button>
+            </div>
```
Comment (i-vyshnevska) [inline base_contract/views/agreement.xml:17]: can you keep active button

Comment (alexis-via) [reply base_contract/views/agreement.xml:17]: In v13, the active button is replaced by "Action > Archive". You can have a look at the partner form or the product form: there is not Active button any more, you must use the Action menu.

Comment (i-vyshnevska) [reply base_contract/views/agreement.xml:17]: ah, thanks for the explanation, didn't see that

**Pull request merged**